Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm

```diff
@@ -299,6 +302,14 @@ static double cameraZoomScaleFactor(AVCaptureDeviceType deviceType)
     m_lastFramesCount = 0;
 }
 
+#if PLATFORM(IOS_FAMILY)
+void AVVideoCaptureSource::startupTimerFired()
+{
+    if (m_shouldCallNotifyMutedChange)
+        notifyMutedChange(!m_isRunning);
+}
```

Comment:
Should you also set `m_shouldCallNotifyMutedChange = false` here or in `notifyMutedChange`?

Reply:
Updated, thanks!